echidna/core/spectra.py

```diff
@@ -77,6 +80,7 @@ def fill(self, energy, radius, time, weight=1.0):
         radial_bin = (radius - self._radial_low) / (self._radial_high - self._radial_low) * self._radial_bins
         time_bin = (time - self._time_low) / (self._time_high - self._time_low) * self._time_bins
         self._data[energy_bin, radial_bin, time_bin] += weight
+        self._raw_events += 1
```

Comment:
Will do but I think this is still the correct thing to do. Even if the weight is 2 you are still using 1 _raw_event to generate these 2 events in the output spectra